packages/lib/test/contracts/upgradeability/AdminUpgradeabilityProxy.behaviour.js

```diff
@@ -53,6 +54,21 @@ export default function shouldBehaveLikeAdminUpgradeabilityProxy(
     this.proxyAddress = this.proxy.address;
   });
 
+  afterEach(function () {
+    sinon.restore();
+  });
+
+  it('uses the correct admin storage slot', async function() {
```

Comment:
Note that this test does not check that the Proxy class falls back to the deprecated label. I'd suggest adding a `LegacyProxy` contract in the mocks folder, and testing against it to ensure both the storage and admin are correctly retrieved.

Reply:
Yeap, I'm working on it. I created a `zos` folder inside the `mock` contracts folder and added the `Zos` prefix to the upgradeability contract names. Is that fine?